samples/user-interface/predictiveback/src/main/java/com/example/platform/ui/predictiveback/PBAnimation.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.platform.ui.predictiveback
+
+class PBAnimation(val title: String, val description: String)
+
+val pbAnimations = mutableListOf<PBAnimation>().apply {
```

Comment:
it's better to use the constructor directly

```
mutableListOf(PBAnimation(...), PBAnimation(..), ...)
```

And if this list is not modified then defnie the variable type to be the immutable version List<PBAnimation>

Reply:
acknowledged